src/mesh_tally.cpp

```diff
@@ -134,9 +135,11 @@ void MeshTally::record_generation(double multiplier) {
       tally_avg[i] = avg;
 
       // Get new variance
-      double var = tally_var[i];
-      var = var + ((val - old_avg) * (val - avg) - (var)) * invs_dg;
-      tally_var[i] = var;
+      if (g > 1) {
+        double var = tally_var[i];
+        var = var + ((val - old_avg) * (val - avg) - (var)) * invs_dg;
+        tally_var[i] = var;
```

Comment:
This is no longer correct, and is different from what is on `feature/operators'. We need to make sure that this version does not get merged. This appears to be the biased variance estimate.